com.mojang.Minecraft.json

```diff
@@ -10,19 +10,6 @@
     "tags": [
         "proprietary"
     ],
-    "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk11"
-    ],
-    "add-extensions": {
-        "org.freedesktop.Sdk.Extension.openjdk8": {
```

Comment:
Is it possible to still use a custom JDK with the new client? If so, removing this is not a good idea - it is an optional extension for some old mods and other things that do not work with new JDK.

Reply:
Oops forgot to comment! Minecraft downloads the necessary version itself. 
![image](https://user-images.githubusercontent.com/45434286/123548229-36249280-d764-11eb-8729-b57158823b39.png)

Reply:
Ah! :-)